src/riak_kv_entropy_manager.erl

```diff
@@ -116,16 +124,27 @@ release_lock(Pid) ->
 %%      will try to acquire a concurrency lock. If successsful, the request is
 %%      then forwarded to the relevant index_hashtree to acquire a tree lock.
 %%      If both locks are acquired, the pid of the remote index_hashtree is
-%%      returned.
+%%      returned. This function assumes an undefined version of the hashtree
```

Comment:
Instead of stating what the function does, I feel like this comment would be more helpful if it explained why it does it. Something like "this is left over for compatibility in mixed clusters with pre-2.2 nodes, from before we added versioning to the hashtree."